### PR TITLE
[Cl-10] Make badge inline

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

Make badge an inline element to hopefully improve compatibility with buttons and other elements.

#### Inline block (current)
image

image

image

#### Inline (new)
image

image

image

## Before you submit

```
- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

**Comment by Hinton** [comment]
@danielleflinn This change seems to make the badges behave a bit better in respect to other elements on the pages.

File: libs/components/src/badge/badge.directive.ts [hunk 1]
```diff
@@ -26,11 +26,10 @@ const hoverStyles: Record<BadgeTypes, string[]> = {
 export class BadgeDirective {
   @HostBinding("class") get classList() {
     return [
-      "tw-inline-block",
+      "tw-inline",
```
Comment (Hinton) [inline libs/components/src/badge/badge.directive.ts:29]: Should we explicitly set the type here, or rely on the correct display level of the actual component? I.e. buttons should perhaps still be block.

**Comment by coroiu** [comment]
Changing to `inline` but leaving `padding` at `0.25rem` results in a bit bigger badges.
An alternative would be to change `padding` to `0.125rem`, resulting in basically unchanged appearance.

#### Inline block (current)
image

#### Inline `padding-top: 0.25rem; padding-bottom: 0.25rem;`
image

#### Inline `padding-top: 0.125rem; padding-bottom: 0.125rem;`
image

File: libs/components/src/badge/badge.directive.ts (hunk 1 above)
Comment (coroiu) [reply libs/components/src/badge/badge.directive.ts:29]: Absolutely no idea, seems like sliighty more risk to rely on correct display level of the actual component?

**Comment by Hinton** [comment]
@danielleflinn thoughts on the design aspects for padding?

**Comment by danielleflinn** [comment]
.125rem looks good to me, let's go ahead and update. @coroiu thanks for the suggestion!

File: libs/components/src/badge/badge.directive.ts (hunk 1 above)
Comment (Hinton) [reply libs/components/src/badge/badge.directive.ts:29]: Oh well, if it breaks I blame you.

**Review by Hinton — APPROVED** [review]

**Pull request merged**